aten/src/ATen/native/cuda/UnaryGeometricKernels.cu

```diff
@@ -131,15 +158,42 @@ void atanh_kernel_cuda(TensorIteratorBase& iter) {
       });
 }
 
+const char tan_name[] = "tan_kernel";
```

Comment:
nit for the future - currently `_kernel` is unconditionally added to the kernel name by codegen, so this kernel will show up in the profile as `tan_kernel_kernel`. If possible, don't add `kernel` to names in ops (this is a preexisting condition for a lot of existing kernels, so no need to fix now).

Reply:
Hi, @ngimel!
Noted. I'll keep that in my mind. Thanks!